tests/bearlib/languages/documentation/DocBaseClassTest.py

```diff
@@ -4,25 +4,31 @@
     DocstyleDefinition)
 from coalib.bearlib.languages.documentation.DocumentationComment import (
     DocumentationComment)
-from coalib.bearlib.languages.documentation.DocumentationExtraction import (
-    extract_documentation)
+from coalib.bearlib.languages.documentation.DocBaseClass import (
+    DocBaseClass)
 from tests.bearlib.languages.documentation.TestUtils import (
     load_testdata)
+from coalib.results.SourceRange import SourceRange
 from coalib.results.TextPosition import TextPosition
 
 
-class DocumentationExtractionTest(unittest.TestCase):
+BaseDoc = DocBaseClass()
```

Comment:
I need an instance of DocBaseClass to test it, just like in case of LocalBear. hmm need to think about different name though.